### DISCUSSION TITLE
How to extends?

### DISCUSSION THREAD

**Discussion by yurkimus** [prisma/prisma#5403] [Q&A] [answered]
I have models where `id`, `createdAt` and `updatedAt` are common fields.

Usually I create base `Entity` and pass to it common fields.
```
model Entity {
  ... some common fields ...
}
```

Implementation:
```
model User extends Entity {
  ... includes Entity fields ...
  &
  ... something for user ...
}
```

How I can do it with Prisma?

**Comment by ryands17** [comment, marked as answer]
@AngryBotIY 👋 

Unfortunately this isn't possible with Prisma at the moment. It would be great if you could add a 👍 to this feature request https://github.com/prisma/prisma/issues/1291 so that we can look into the priority for this :)

**Reply by yurkimus** [reply]
👋👋👋 Thanks! I did 👍 :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
